### PR TITLE
feat(tutorial): complete step 3

### PR DESCRIPTION
**Pull request by timoken**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/content/RepoPage/RepoPage.js [hunk 1]
```diff
@@ -28,45 +65,81 @@ const headers = [
...
-    createdAt: 'Date',
-    updatedAt: 'Date',
-    issueCount: '123',
-    stars: '456',
-    links: 'Links',
-  },
-];
-
 const RepoPage = () => {
+  const LinkList = ({ url, homepageUrl }) => (
```
Comment (kellychurchill) [inline src/content/RepoPage/RepoPage.js]: `LinkList` and `getRowItems` should be declared outside the `RepoPage` const declaration. This is making your formatting off.

Comment (timoken) [reply src/content/RepoPage/RepoPage.js]: Fixed https://github.com/carbon-design-system/carbon-tutorial/pull/901/commits/67cfb920cd743471a80dc6ec7c74c150fc887e22

**Comment by kellychurchill** [comment]
Hi! Your repo page still does not match the preview page: https://react-step-4--carbon-tutorial.netlify.com/#/repos

You're missing the padding around the table. It looks like you deleted the wrappers when you added the Query block: 
```
    <div className="bx--grid bx--grid--full-width bx--grid--no-gutter repo-page">
      <div className="bx--row repo-page__r1">
        <div className="bx--col-lg-16">
```

**Comment by timoken** [comment]
> Hi! Your repo page still does not match the preview page: https://react-step-4--carbon-tutorial.netlify.com/#/repos
> 
> You're missing the padding around the table. It looks like you deleted the wrappers when you added the Query block:
> 
> ```
>     <div className="bx--grid bx--grid--full-width bx--grid--no-gutter repo-page">
>       <div className="bx--row repo-page__r1">
>         <div className="bx--col-lg-16">
> ```

Ah, yes you're right! I've added it back here: https://github.com/carbon-design-system/carbon-tutorial/pull/901/commits/9140bacab8d168aaf021ae81c5313e99cb56ed59

**Comment by kellychurchill** [comment]
👍 Looks good - 🎉 You are one step closer to completion of the Carbon React tutorial and earning the digital credential.

**Pull request closed without merging**